app.R

```diff
@@ -0,0 +1,1123 @@
+                        choices = c('Age',
+                                    'Height',
+                                    'Pre-diet Weight'), 
+                        selected = 'Age'
+                      ),
+                      radioButtons(
+                        inputId = 'select_covar', 
+                        label = 'Select Covariance',
+                        inline = TRUE,
+                        choices = c('Gender',
+                                    'Diet'), 
+                        selected = 'Gender'
+                      )
+                    ),
+                    conditionalPanel(
+                      condition = "input.menu1 == 'Random'",
+                      sliderInput(
+                        inputId = 'slope1',
+                        label = 'Change the slope of Line A',
+                        min = -5,
+                        max = 5,
+                        value = 0,
+                        step = 1)
+                      ,
+                      sliderInput(
+                        inputId = 'slope2',
+                        label = 'Change the slope of Line B',
+                        min = -5,
+                        max = 5,
+                        value = 0,
+                        step = 1
+                      ),
+                      sliderInput(
+                        inputId = 'inter1',
+                        label = 'Change the intersection of Line A',
+                        min = -5,
+                        max = 5,
+                        value = 0,
+                        step = 1
+                      ),
+                      sliderInput(
+                        inputId = 'inter2',
+                        label = 'Change the intersection of Line B',
+                        min = -5,
+                        max = 5,
+                        value = 0,
+                        step = 1
+                      ),
+                      sliderInput(
+                        inputId = 'sample',
+                        label = 'Change the sample size',
+                        min = 100,
+                        max = 800,
+                        value = 100,
+                        step = 50
+                      )
+                    ),
+                    fluidRow(
+                      uiOutput('p'),
+                      align = "left"),
+                    conditionalPanel(
+                      condition = "input.menu1 != 'Random'",
+                      downloadButton(
+                        outputId = "downloadData", 
+                        label = "Download the Dataset",
+                        icon = shiny::icon("download")
+                      )
+                    )
+                  ),
+                  
+                  mainPanel(
+                    plotOutput('plot_gg'),
+                    tags$b(verbatimTextOutput('analysis1')),
+                    div(style = "text-align: center",
```

Comment:
Because the button is in the main panel, it does not show up centered on the page. I think you can try removing it from the main panel and just have in the page itself so it is centered!